Lombiq.HelpfulLibraries.Samples/Navigation/HelpfulLibrariesNavigationProvider.cs

```diff
@@ -20,7 +20,7 @@ protected override void Build(NavigationBuilder builder)
         var context = _hca.HttpContext;
         builder
             .Add(T["Helpful Libraries"], builder => builder
-                .Add(T["LINQ to DB"], subMenu => subMenu.Url("#").AddClass("disabled menuWidget__link_title"))
+                .AddLabel(T["LINQ to DB"])

```

Comment:
This looks more like a disabled menu item than a label.

Reply:
We want to add labels because Bootstrap menu doesn't support multiple submenu layers. The fact that it's done by add a "disabled" class is just an implementation detail that the library consumer doesn't need to know or care about.

Reply:
My problem is that by default, this menu item looks like a disabled one, instead of conveying the idea that it's a label.
![image](https://user-images.githubusercontent.com/1976647/167020608-c2c309d7-e9ea-424e-a280-32b32f3b9df8.png)

Talking about UX, not implementation detail. Regardless of how this is implemented, the label should be display (by default) in a way that it's not confusing.

Reply:
It's not confusing for me tho. :) What do you suggest?

Reply:
Just a hint: If you want an entry in the list that's not clickable, you can use the `pointer-events: none` style.

Reply:
I'm not good with design, please look up what others do.

Reply:
I need "disabled" class becuase otherwise Bootstrap marks any link as "active" on the home page that doesn't have a href, has an empty href, or a href only containing "#". It causes this weird behavior:
![image](https://user-images.githubusercontent.com/4007293/167026286-b99f73d5-beaa-47aa-8ec3-6bdf65bb6eea.png)

Reply:
I put the "disabled" back, but added some styling to Lombiq.BaseTheme to undo the grey looks. Also made it bold in Lombiq.BaseTheme.Samples to look better (that felt too opinionated to put into the base theme).
![image](https://user-images.githubusercontent.com/4007293/167032473-1100ac17-51bb-47ef-b841-4e5cd8cb41fc.png)

Reply:
Looks good!